jbehave-core/src/main/java/org/jbehave/core/model/ExamplesTableFactory.java

```diff
@@ -101,7 +102,18 @@ public void useKeywords(Keywords keywords) {
         this.keywords = keywords;
     }
 
+    public void useExampleTableMonitor(ExampleTableMonitor exampleTableMonitor) {
```

Comment:
`ExamplesTableFactory` should be immutable, so such configuration should be moved to `Configuration` or any similar controls